manifests/server.pp

```diff
@@ -569,7 +574,7 @@
   if $ca {
     $ssl_ca_cert = "${ssl_dir}/ca/ca_crt.pem"
     $ssl_ca_crl  = "${ssl_dir}/ca/ca_crl.pem"
-    $ssl_chain   = "${ssl_dir}/ca/ca_crt.pem"
+    $ssl_chain   = pick($ssl_chain_filepath, "${ssl_dir}/ca/ca_crt.pem")
```

Comment:
The default should be given in params.pp rather than using pick() here, it'll be easier for users and tools like kafo (for foreman-installer) to see the default value.